tests/conftest.py

```diff
@@ -28,5 +28,5 @@ def tmp_contracts(tmpdir):
 
 
 @pytest.fixture
-def deployer(twig_deployer):
-    return twig_deployer(CONTRACTS_DIR)
+def deployer(vy_deployer):
```

Comment:
I think I understand a little better what's going on here and with the other `pytest-ethereum` PR which I commented on which adds `twig` as a dependency.

Maybe we should schedule a call to try and sort this out.

Reply:
Cool, just saw this - so disregard the comment I just left on the other pr. We have our 1-on-1 tomorrow, we can discuss this then?